GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -348,6 +357,7 @@ public void Add(string aId, string[] aParentIds, DataType aType, GitRevision aDa
             UpdateData();
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity")]
```

Comment:
:bulb: I'm not convinced this is correct. I would prefer to see it moved to its own pull request and evaluated separately.